tests/test-component.php

```diff
@@ -83,4 +82,71 @@ public function test_name() {
 		$this->assertEquals( '', $this->get_component( 'example-001' )->get_namespace(), 'Expected an empty string.' );
 		$this->assertEquals( 'irving', $this->get_component( 'irving-namespace' )->get_namespace() );
 	}
+
+	/**
+	 * Test the children functionality.
+	 *
+	 * Methods to test:
+	 * get_children()
+	 * set_children()
+	 * prepend_children()
+	 * append_children()
+	 * set_child()
+	 * prepend_child()
+	 * append_child()
+	 * sanitize_children()
+	 */
+	public function test_children() {
+
+		$parent = $this->get_component( 'children-test-001' );
+		$child  = $this->get_component( 'children-test-002' );
+
+		// get_children()
```

Comment:
Good to note these as todos. When implementing, each test should be handled in a separate test method rather than testing all of these within a single method.